packages/react/src/components/Tile/Tile.js

```diff
@@ -360,7 +363,7 @@ export function ExpandableTile({
   const aboveTheFold = useRef(null);
   const belowTheFold = useRef(null);
   const tileContent = useRef(null);
-  const tile = useRef(null);
+  const tile = useRef(ref);
```

Comment:
This won't work. The refs need to be merged - we have a helper function you can use for it

```
const tile = useRef(null);
const ref = useMergedRefs([forwardRef, tile]);
```

What I would recommend is this:
- Rename the forward ref `ref` to `forwardRef`
- Use the above snippet
- Apply `ref` to the DOM element instead of `tile` - `ref={ref}`